index.html

```diff
@@ -0,0 +1,238 @@
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-card">
+                        <div class="main-card-img">
+                            <i class="fa fa-rocket"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                PANACCS Exhibition
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Appreciate the various contributions of industry leaders for the revolutonary   cloud domain.                    
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-card">
+                        <div class="main-card-img">
+                            <i class="fa fa-user"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                Forum
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Have the time to share your thoughts and opinions with experts for each topic.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-card">
+                        <div class="main-card-img">
+                            <i class="fa fa-microphone"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                Workshop
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Try creating your own work using open source license rather than just watching at it.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-card">
+                        <div class="main-card-img">
+                            <i class="fa fa-cogs"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                PANACCS Ignite
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Get opportunities to network with PANACCS affiliates around the world, also after the summit.
+                            </p>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] I think it would be better if you add interaction on your **Main Program Cards** when one of the cards has hovered and **use the correct color on each card title(i.e.: `#ec5242`)** to stick to the template design.
![Screenshot from 2022-09-16 07-02-52](https://user-images.githubusercontent.com/83042742/190560675-2fea890b-c9a6-4bca-bc0a-9349537c1de9.png)